pvlib/test/test_singlediode.py

```diff
@@ -167,8 +170,19 @@ def get_pvsyst_fs_495():
             },
             (5e-4, 0.04)
         ),
-        (POA, TCELL, {'pmp': 76.26, 'isc': 1.387, 'voc': 79.29}, (1e-3, 1e-3))]
-)  # DeSoto @(888[W/m**2], 55[degC]) = {Pmp: 72.71, Isc: 1.402, Voc: 75.42)
+        # other conditions
```

Comment:
curious: why the format change here

Reply:
To make it easier for posterity to parse it visually.